src/components/cms/BrowseSkill/skillLoader.js

```diff
@@ -34,11 +34,43 @@ const addLoader = () => {
 
 const SkillLoader = () => {
   return (
```

Comment:
Call isMobileView only once here and store it in a variable. It will minimize the function calls. Also, can you fix the card loader in the mobile view?

Reply:
> Call isMobileView only once here and store it in a variable. It will minimize the function calls.

Done sir!

Reply:
> Also, can you fix the card loader in the mobile view?

Yes I have noticed that in mobile view those cards are not as good as they need to be
I have opened an issue #3209 related to it and soon will fix it